config.json

```diff
@@ -5,9 +5,13 @@
   "active": true,
   "deprecated": [
     "accumulate",
+    "binary",
     "bottles",
     "counter",
-    "point-mutations"
```

Comment:
I regret it, but since

1. trailing comma is not allowed
2. we keep this alphabetical

it was necessary.

There is no rule saying we have to keep it alphabetical, but it started out that way and I didn't think commit cleanliness was a sufficient reason to break it. Unless y'all think it is, in which case I will gladly!